ignite/engine/events.py

```diff
@@ -130,12 +130,16 @@ def wrapper(engine: "Engine", event: int) -> bool:
         return wrapper
 
     @staticmethod
-    def once_event_filter(once: int) -> Callable:
+    def once_event_filter(once: Union[int, list]) -> Callable:
```

Comment:
Better approach is to make `once_event_filter` working on lists instead of int or list. Before calling this function you can convert int value into the list with 1 value : `once = [once, ]`.

The point is to simplify two if conditions into one condition: `event in once`.